src/Sulu/Component/Rest/RequestParametersTrait.php

```diff
@@ -58,12 +62,13 @@ protected function getRequestParameter(Request $request, $name, $force = false,
      */
     protected function getBooleanRequestParameter($request, $name, $force = false, $default = null)
     {
+        /** @var mixed|bool $value */
         $value = $this->getRequestParameter($request, $name, $force, $default);
         if ('true' === $value || true === $value) {
             $value = true;
         } elseif ('false' === $value || false === $value) {
             $value = false;
-        } elseif ($force && true !== $value && false !== $value) {
```

Comment:
revert the changs inside this trait to avoid regressions. the trait should be avoided as we will remove it in 3.0.